app/controllers/public/api/v1/inboxes/conversations_controller.rb

```diff
@@ -1,15 +1,31 @@
 class Public::Api::V1::Inboxes::ConversationsController < Public::Api::V1::InboxesController
   include Events::Types
-  before_action :set_conversation, only: [:toggle_typing, :update_last_seen]
+  before_action :set_conversation, only: [:toggle_typing, :update_last_seen, :show, :resolve]
 
   def index
     @conversations = @contact_inbox.hmac_verified? ? @contact.conversations : @contact_inbox.conversations
   end
 
+  def show; end
+
   def create
     @conversation = create_conversation
   end
 
+  def resolve
```

Comment:
let's use the `toggle_status`  name for this endpoint as well to ensure consistency. Since we have the same name for `application` and `widget` apis.

Reply:
Should we only rename the method or do we also adapt the functionality to be closer to a `toggle` than a `resolve` ?

Reply:
Thank you for clearing this up. I will leave the method as is and only rename it.